presto-rcfile/src/main/java/io/prestosql/rcfile/TimestampUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.rcfile;
+
+import io.prestosql.spi.block.Block;
+import io.prestosql.spi.type.LongTimestamp;
+import io.prestosql.spi.type.TimestampType;
+
+import java.time.LocalDateTime;
```

Comment:
Avoid java.time's objectful representation.
Can we use LongTimestamp or something like DecodedTimestamp (rename to HiveTimestamp?)

Reply:
I use this to format a timestamp with precision > milliseconds.  Since Joda only supports millisecond precision, I have to use `java.time`.